# HG changeset patch
# User Rik <rik@octave.org>
# Date 1479926334 28800
#      Wed Nov 23 10:38:54 2016 -0800
# Branch stable
# Node ID 9cb15c322464352376dc1dbef7d3e89bdbf36b83
# Parent  9d7026337bea5d691f35df20aae3553f14c9c191
doc: Add programming note to caxis docstring.

* caxis.m: Add programming note to docstring explaining color axis.

diff --git a/scripts/plot/appearance/caxis.m b/scripts/plot/appearance/caxis.m
--- a/scripts/plot/appearance/caxis.m
+++ b/scripts/plot/appearance/caxis.m
@@ -34,17 +34,23 @@
 ## and @var{cmax} to the maximum data value.  If @qcode{"manual"} is specified
 ## then the @qcode{"climmode"} property is set to @qcode{"manual"} and the
 ## numeric values in the @qcode{"clim"} property are used for limits.
 ##
 ## If the first argument @var{hax} is an axes handle, then operate on
 ## this axis rather than the current axes returned by @code{gca}.
 ##
 ## Called without arguments the current color axis limits are returned.
-## @seealso{colormap}
+##
+## Programming Note: The color axis affects the display of image, patch, and
+## surface graphics objects, but @strong{only} if the @qcode{"cdata"} property
+## has indexed data and the @qcode{"cdatamapping"} property is set to
+## @qcode{"scaled"}.  Graphic objects with true color @code{cdata}, or
+## @qcode{"direct"} @code{cdatamapping} are not affected.
+## @seealso{colormap, axis}
 ## @end deftypefn
 
 function limits = caxis (varargin)
 
   [hax, varargin, nargin] = __plt_get_axis_arg__ ("caxis", varargin{:});
 
   oldfig = [];
   if (! isempty (hax))
